week-5/C-promises/exercise.js

```diff
@@ -72,8 +79,13 @@ function exercise5() {
   Take the result of your promise and insert the it into the #exercise6
   element
 */
+const exe6Promise = async () => { 
+  throw new Error ("Something went wrong!"); // this will return a promise!
```

Comment:
This works too! Do you know the other way of creating a rejected Promise?

Reply:
Yes, I do. It is similar to what you did in the previous exercises.

Reply:
const exe6Promise = () =>
  new Promise((resolve, reject) => reject("Something went wrong!"));